src/Database/Builder.php

```diff
@@ -14,6 +14,8 @@
  */
 class Builder extends BuilderModel
 {
+    use \October\Rain\Database\Eloquent\Concerns\QueriesRelationships;
```

Comment:
This should be namespaced to `\October\Rain\Database\Concerns\QueriesRelations;` (remove the Eloquent)

Reply:
I've moved the file and modified the namespace